test/jobs/zendesk_job_test.rb

```diff
@@ -0,0 +1,61 @@
+require_relative '../test_helper'
+
+SingleCov.covered!
+
+describe 'Zendesk Job' do
+  hostname = 'https://hostname.com'
+  username = 'username@email.com'
+  password = '123456'
+  describe 'boostraps zendesk job' do
+    it 'bootstraps by getting environment variables' do
+      ENV['ZENDESK_HOSTNAME'] = hostname
+      ENV['ZENDESK_USERNAME'] = username
+      ENV['ZENDESK_PASSWORD'] = password
```

Comment:
this will leave these variables set for the rest of the test suite, leading to possible race conditions if you add a negative test asserting nothing happens when they aren't set.